.github/workflows/chromatic.yml

```diff
@@ -1,6 +1,9 @@
 name: Chromatic
 
-on: push
+on:
+  push:
+    branches:
+      - main
```

Comment:
I think we're letting chromatic create builds for those merge requests that need validation from the design team before merging to main. Do you think we can remove this restriction?